lib/pa_ess/utilities.ex

```diff
@@ -691,7 +691,7 @@ defmodule PaEss.Utilities do
   def audio_take(:stop_away), do: "535"
   def audio_take(:stops_away), do: "534"
   def audio_take(:the_first), do: "866"
-  def audio_take(:scheduled_to_arrive_at), do: "865"
+  def audio_take(:departs_at), do: "927"
```

Comment:
Already synthesized and uploaded I assume?

Reply:
Yes!